<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pac</class>
 <widget class="QDialog" name="pac">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>429</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="2">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Дата</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Время</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Город</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Название соревнования</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Вид спорта</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="6" column="5">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Критерий поиска</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QTableView" name="tableView">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Обновить расписание</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Работы с базой данных</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="4">
    <widget class="QListWidget" name="listWidget_2">
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Информация о соревновании</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Название соревнований</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QListWidget" name="listWidget">
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
